Mods/CombatExtended/Defs/StuffCategoryDefs/StuffCategories.xml

```diff
@@ -1,6 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Defs>
 
+	<StuffCategoryDef>
```

Comment:
In reguards to this, i would actually prefer not to include this, or add a patch to remove this and replace any that use it, with a HSK one

Reply:
Refering to "Steeled"

Reply:
Found 8 ThingDefs has this StuffCategoryTag:
- Steel
- Plasteel
- Apparel_SimpleHelmet
- Apparel_FlakVest
- Apparel_KidHelmet
- UnfinishedSteeledTechArmor
- CE_Apparel_MeleeShield
- CE_Apparel_BallisticShield

Reply:
Comment out in commit 55126d5